### ISSUE TITLE
Create separate preferences for 4.0 and 3.3

### ISSUE THREAD

**Issue by jenhicks** [opensim-org/opensim-gui#917]

**Comment by carmichaelong** [comment]
Preferences box is much cleaner now on W10 71c8c385-2018-08-11. Mine looks as follows:

![opensimpref](https://user-images.githubusercontent.com/7455790/44059375-fe05d34e-9f05-11e8-95bc-413b25af7590.JPG)

@aymanhab Comparing with the picture and discussion from #948, I may be missing the "Visualizer: Muscle Color Scheme" line. Please let me know if there indeed should be another line or if this is correct as is.

**Comment by aymanhab** [comment]
@carmichaelong The preference of color scheme is created on demand when loading a model with muscles. If you still don't see it then I will revisit. Thank you.

**Comment by carmichaelong** [comment]
Ah, thanks. Popped up with a model:

![capture](https://user-images.githubusercontent.com/7455790/44060213-c7292120-9f08-11e8-8084-852e26242c7e.JPG)

**Comment by chrisdembia** [comment]
Verifying with OpenSim 71c8c38-2018-08-11 on Mac:

<img width="532" alt="image" src="https://user-images.githubusercontent.com/846001/44113233-3f0d75d4-9fbc-11e8-9f0d-ccf9ca7949b1.png">

1. The preferences window is not large enough by default to fit the full text of the `Option` column.
2. What is the `Paths: Scripts Path` option? Is it necessary? I don't know of any `Scripts` folder in 4.0. I would think this path is `<Resources-Dir>/Code/GUI`.
3. > The preference of color scheme is created on demand when loading a model with muscles (and animating result files with states/colors). If you still don't see it then I will revisit. Thank you.

It would be preferable for the `Visualizer: Muscle Color Scheme` to appear always, not only after loading a model.

4. The default value for `Experimental Marker Radius` has a `.0` but the default value for `Muscle Display Radius does not. I think we should be consistent about the `.0`.

**Comment by aymanhab** [comment]
Thanks for the feedback @chrisdembia 
1. Yes we can make it wider but it will likely look sparse since some names are short while others are now much longer,Users can make the trade-off by sliding the column separator.
2. The Paths: Scripts is supposed to be the folder that the GUI uses to locate scripts on first launch but it will change with user selection. It may actually be unnecessary, would be good to hear feedback from @jimmyDunne and others testing the GUI scripts if helpful.
3. Yes we can do that, though you could argue  a user that never uses muscles shouldn't care.
4. These preferences are stored as Strings that get parsed. You can enter 5, 5.0, 5.00000 and it will stay the same, the code will parse it and will not modify it.

**Comment by chrisdembia** [comment]
> The Paths: Scripts is supposed to be the folder that the GUI uses to locate scripts on first launch but it will change with user selection. It may actually be unnecessary, would be good to hear feedback from @jimmyDunne and others testing the GUI scripts if helpful.

This preference doesn't seem to have an effect to me, or to be updated when I open a script.

> These preferences are stored as Strings that get parsed. You can enter 5, 5.0, 5.00000 and it will stay the same, the code will parse it and will not modify it.

That matches my understanding. But I'm talking about consistency with the default value.

**Comment by aymanhab** [comment]
I think the rationale for Paths: Scripts was so that you have some scripts available in cascade menu on launch without "browsing", if that doesn't seem to be the case (hard to test on Windows since Preferences do persist) then we should re-evaluate.

The default value for Experimental Marker Size will be "10" without a period with PR to address issue #643 so we should be good after it goes in.

**Comment by chrisdembia** [comment]
@aymanhab thanks for your responses regarding (2) and (4). I've created a new issue for (2).

(3) is somewhat minor.

I would like (1) to be addressed before marking this issue as verified.

**Comment by chrisdembia** [comment]
Thanks @aymanhab I moved this to the VWM pipeline.
